components/avalancheCenterList.ts

```diff
@@ -0,0 +1,60 @@
+import {AvalancheCenter, AvalancheCenterID, avalancheCenterIDSchema} from 'types/nationalAvalancheCenter';
+
+export interface AvalancheCenterListData {
+  center: AvalancheCenter;
+  blurb?: string;
+}
+
+const supportedAvalancheCenters: AvalancheCenterID[] = ['NWAC', 'SNFAC'];
+
+export enum AvalancheCenters {
+  SupportedCenters,
+  AllCenters,
+}
+
+// In order to display data for the center, we need to know about it so that we have a logo, etc.
+export const filterToKnownCenters = (ids: string[]): AvalancheCenterID[] => {
+  const knownCenters: AvalancheCenterID[] = [];
+  for (const center of ids) {
+    const idResult = avalancheCenterIDSchema.safeParse(center);
+    if (idResult.success) {
+      knownCenters.push(idResult.data);
+    }
+  }
+  return knownCenters;
+};
+
+const blurbs: Record<AvalancheCenterID, string> = {
```

Comment:
I imagine there's some way to indicate on the type level that this record has *some* but not all entries, but as written `tsc` yelled at me and made me add in every possible key.